### PR TITLE
Remove flag to enable native cosmetic filtering version

### PR DESCRIPTION
**Pull request by antonok-edm**
The `BraveAdblockCosmeticFilteringNative` feature was removed with https://github.com/brave/brave-core/pull/9995 which landed in 1.31.x, so `NativeCosmeticFilteringStudy` is a no-op on 1.31.x and up.

### CODE REVIEW

**Review by bsclifton — APPROVED** [review]

**Comment by kjozwiak** [comment]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR as per https://github.com/brave/brave-variations/pull/324#issuecomment-1198832996.

**Pull request merged**
